pkgs/applications/audio/proxtx-clipper/default.nix

```
@@ -0,0 +1,53 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, cmake
+, libopus
+, pkg-config
+, stdenv
+, darwin
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "proxtx-clipper";
+  version = "unstable-2024-02-17";
+
+  src = fetchFromGitHub {
+    owner = "Proxtx";
+    repo = "clipper";
+    rev = "782c1d72232b095fdd9d4b55e92e6afbcbc95575";
+    hash = "sha256-prJRe5l4WaX2gAdZ6QCDb2VHqvIUUT1Fi5je3L0Rxj8=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+  };
+
+  postPatch = ''
+    ln -s ${./Cargo.lock} Cargo.lock
+  '';
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = [
+    libopus
+  ] ++ lib.optionals stdenv.isDarwin [
+    darwin.apple_sdk.frameworks.Security
+  ];
+
+  #Change the binary name to from clipper to proxtx-clipper
+  postInstall = ''
+    mv $out/bin/clipper $out/bin/proxtx-clipper
+  '';
+
+  meta = with lib; {
```

```suggestion
  meta = {
```

Better to avoid `with lib`